src/Illuminate/Validation/ValidationData.php

```diff
@@ -7,6 +7,11 @@
 
 class ValidationData
 {
+    /**
```

Comment:
please add a short description, and format the phpdoc like the rest of the framework

Reply:
fixed